### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by darrenabelaquaq**
update readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,30 +122,34 @@ e.g `"query" : "([]a:enlist hello world)"` Failure response (query should have q
...
-    java -jar -Dspring.profiles.active=test -Dspring.config.location=C:\Programming\application-test.properties target\q-REST-1.1-SNAPSHOT.jar
+3. Run following command from command prompt (ensure you select your application.properties file created in step 1):
 
+       java -jar -Dspring.profiles.active=test -Dspring.config.location=C:\Programming\application-test.properties target\q-REST-1.1-SNAPSHOT.jar
+       
 * -Dspring.profiles.active would be the profile you wish to run the application with which would have its own `application-{profileName}.properties` file. 
 
 * -Dspring.config.location would be the location of the custom properties file.
 
+Alternatively, you can download the source code from git hub, update the project with your changes, run a maven build, then run project from your chosen ide by executing the main springboot application class as seen within project.
```
Comment (jonathonmcmurray) [inline README.md]: git hub -> GitHub
ide -> IDE

not sure but I think springboot -> SpringBoot

File: README.md [hunk 2]
```diff
@@ -122,30 +122,34 @@ e.g `"query" : "([]a:enlist hello world)"` Failure response (query should have q
...
-Either pull the code from github, update `application.properties` and maven clean install to create the new jar then run:
-
-    java -jar target\q-REST-1.1-SNAPSHOT.jar
+1. Configure your own application.properties file to point to your specific kdb instance:
+    
+    kdb.host=localhost
+    kdb.port=6007
+    kdb.username=admin
+    kdb.password=admin
+    server.port=8080
```
Comment (jonathonmcmurray) [inline README.md]: code blocks in gh-flavoured markdown should have ` ``` ` before & after them to ensure they render properly on GitHub

File: README.md [hunk 3]
```diff
@@ -122,30 +122,34 @@ e.g `"query" : "([]a:enlist hello world)"` Failure response (query should have q
...
 
+       java -jar -Dspring.profiles.active=test -Dspring.config.location=C:\Programming\application-test.properties target\q-REST-1.1-SNAPSHOT.jar
+       
 * -Dspring.profiles.active would be the profile you wish to run the application with which would have its own `application-{profileName}.properties` file. 
 
 * -Dspring.config.location would be the location of the custom properties file.
 
+Alternatively, you can download the source code from git hub, update the project with your changes, run a maven build, then run project from your chosen ide by executing the main springboot application class as seen within project.
+
+Project may also be run using a docker, please find a sample Dockerfile defined within project src.
```
Comment (jonathonmcmurray) [inline README.md]: this line seems misleading as Dockerfile is outside of directory named `src`

also not overly familiar with docker, is "run using a docker" standard terminology? I think I've more often seen things like "run as a docker image" or something along those lines?

**Comment by aquaq** [comment]
For now can we simply approve these requests.


James


________________________________

**Pull request merged**